Library/Formula/docker-machine.rb

```diff
@@ -14,35 +14,21 @@ class DockerMachine < Formula
 
   depends_on "go" => :build
 
-  go_resource "github.com/tools/godep" do
-    url "https://github.com/tools/godep.git", :revision => "58d90f262c13357d3203e67a33c6f7a9382f9223"
-  end
-
-  go_resource "github.com/kr/fs" do
-    url "https://github.com/kr/fs.git", :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
-  end
-
-  go_resource "golang.org/x/tools" do
-    url "https://github.com/golang/tools.git", :revision => "473fd854f8276c0b22f17fb458aa8f1a0e2cf5f5"
-  end
+  def install
+    mkdir_p buildpath/"src/github.com/docker"
+    ln_s buildpath, buildpath/"src/github.com/docker/machine"
 
-  go_resource "github.com/docker/machine" do
-    url "https://github.com/docker/machine.git", :revision => "eaff7635fc821d783eb6e996dbdad22dacdd058c"
-  end
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
 
-  def install
-    ENV["GOPATH"] = buildpath
     Language::Go.stage_deps resources, buildpath/"src"
```

Comment:
This can be removed if we are no longer using resources.

To confirm, all of the dependencies are being vendored?